### PR TITLE
Fixes #52: Help text (-h) for memory size (-m) is incorrect.

### LINKED ISSUE #52: Help text (-h) for memory size (-m) is incorrect.
The [help text](https://github.com/mist64/xhyve/blob/master/src/xhyve.c#L138) specifies **-m** as:
> -m: memory size in MB

But [xhyverun.sh](https://github.com/mist64/xhyve/blob/master/xhyverun.sh#L13) indicates that it can also be suffixed with (at least) **G** for gigabyte.

> MEM="-m 1G"

### PR DESCRIPTION
**Pull request by rickard-von-essen**

### CODE REVIEW

**Comment by jeremyhu** [comment]
This was merged into https://github.com/xhyve-xyz/xhyve

**Comment by mist64** [comment]
I'd like to merge this; please fix the conflicts.

**Comment by rickard-von-essen** [comment]
Sorry for the delay now it is fixed!

File: src/xhyve.c [hunk 1]
```diff
@@ -136,6 +136,7 @@ usage(int code)
 		"       -h: help\n"
 		"       -H: vmexit from the guest on hlt\n"
 		"       -l: LPC device configuration\n"
+		"       -m: memory size in MB, may be suffixed with one of K, M, G or T\n"
 		"       -m: memory size in MB\n"
```
Comment (ailispaw) [inline src/xhyve.c]: Here are two lines for `-m`. No deletion?

Comment (xez) [inline src/xhyve.c]: Yeah, that's a mistake.

**Comment by rickard-von-essen** [comment]
:no_good: My sloppy rebase :disappointed:

**Pull request merged**